iotdb-core/node-commons/src/test/java/org/apache/iotdb/commons/utils/TimePartitionUtilsTest.java

```diff
@@ -89,4 +90,19 @@ public void testGetTimePartitionSlot_NegativeBoundaryTime() {
     TTimePartitionSlot actualSlot = TimePartitionUtils.getTimePartitionSlot(testTime);
     assertEquals(expectedSlot.getStartTime(), actualSlot.getStartTime());
   }
+
+  @Test
+  public void testOverflow() {
```

Comment:
Add a test case when timePartitionOrigin is not the default value？

Reply:
In this test class, the timePartitionOrigin has been modified during initialization.

Reply:
i found it `TEST_TIME_PARTITION_ORIGIN`